src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -301,6 +310,8 @@ private bool IsReadyToClose
             }
         }
 
+        public IScope Scope => GetDefaultScope();
```

Comment:
Implemented interface methods can use the inheritdoc tag for the doc comments

Reply:
aware of this already, but I haven't added comments in interfaces. comments will be polished during scopes and collections implementations.